packages/react-native-web/src/exports/Image/types.js

```diff
@@ -102,8 +102,8 @@ export type ImageStyle = {
   tintColor?: ColorValue
 };
 
-export type ImageProps = {
-  ...ViewProps,
+export type ImageProps = {|
+  ...$Exact<ViewProps>,
```

Comment:
Is there a reason you're adding this ([docs ref](https://flow.org/en/docs/types/utilities/#toc-exact)) other than personal pref? I think we're preferring to make this as lean as possible, no?

Reply:
Yeah, I've posted about this in a another comment regarding types: https://github.com/Expensify/react-native-web/pull/11/files#r1081137857